src/components/controls3d/Controls3dDirective.js

```diff
@@ -0,0 +1,60 @@
+goog.provide('ga_controls3d_directive');
+
+(function() {
+
+  var cssRotate = function(element, angle) {
+    var value = 'rotate(' + angle + 'rad)';
+    element.css({
+      '-ms-transform': value,
+      '-webkit-transform': value,
+      'transform': value
+    });
+  };
+
+  var module = angular.module('ga_controls3d_directive', []);
+
+  module.directive('gaControls3d', function() {
+    return {
+      restrict: 'A',
+      templateUrl: 'components/controls3d/partials/controls3d.html',
+      scope: {
+        ol3d: '=gaControls3dOl3d'
+      },
+      link: function(scope, element, attrs) {
+
+        var ol3d = scope.ol3d;
+        var scene = ol3d.getCesiumScene();
+        var camera = scene.camera;
+
+        var tiltIndicator = element.find('.ga-tilt .ga-indicator');
+        var rotateIndicator = element.find('.ga-rotate .ga-indicator');
+
+        camera.moveEnd.addEventListener(function() {
+          scope.$apply(function() {
```

Comment:
I systematically use `$apply` when doing stuff outside angular; no matter what the function does. But it's true that it should not make a big difference here because (as you said) only css is changed.

I will remove it